docs/install.md

```diff
@@ -936,3 +936,155 @@ This procedure can be used to downgrade the active version of SAT.
     the modified CFS configuration. If more than one CFS configuration was
     modified, use the first one.
 
+## Optional: Configuring SAT to run on an external system over the CAN
+
+It is possible to configure SAT so that it can interact with CSM over the CAN. This allows SAT to run from a physical or
+virtual machine which is not a part of the CSM management cluster.
+
+### Limitations
+
+Some SAT subcommands must interact with services which run on the NCNs in the CSM management cluster, including CSM REST
+APIs, Kubernetes, Ceph, and S3. Certain SAT commands, such as `sat bootsys`, may not work as expected.
```

Comment:
I'm not exactly sure what this is trying to say. I think you're trying to convey that some SAT functionality which uses externally accessible APIs will work, while other functionality which depends on functionality local to the management NCNs will not. Here's my suggestion:

```suggestion
Most SAT subcommands work by accessing APIs which are reachable via the CAN. However, certain SAT commands depend on
host-based functionality on the management NCNs and will not work from an external system. This includes the following:

- The `platform-services` and `ncn-power` stages of `sat bootsys`
- The local host information displayed by the `--local` option of `sat showrev`
```

Reply:
I think this is an improvement, but since I didn't do a comprehensive test of all of the SAT functionality, I didn't want an enumerated list to imply that it was exclusive, i.e. that everything else should be expected to work. I suppose a compromise could be to specify `this includes, but may not necessarily be limited to the following:`?